cachix/test/NetRcSpec.hs

```diff
@@ -0,0 +1,59 @@
+module NetRcSpec where
+
+import Paths_cachix
+import Protolude
+import System.Directory (copyFile)
+import System.IO.Temp   (withSystemTempFile)
+import Test.Hspec
+
+import           Cachix.Api (BinaryCache(..))
+import qualified Cachix.Client.NetRc as NetRc
+import           Cachix.Client.Config (Config(..))
+
+
+bc1 :: BinaryCache
+bc1 = BinaryCache
+ { name = "name"
+ , uri = "https://name.cachix.org"
+ , publicSigningKeys = ["pub"]
+ , isPublic = False
+ , githubUsername = "foobar"
+ }
+
+bc2 :: BinaryCache
+bc2 = BinaryCache
+  { name = "name2"
+  , uri = "https://name2.cachix.org"
+  , publicSigningKeys = ["pub2"]
+  , isPublic = False
+  , githubUsername = "foobar2"
+  }
+
+config :: Config
+config = Config
+  { authToken = "token123"
+  , binaryCaches = []
+  }
+
+-- TODO: poor man's golden tests, hspec doesn't have a lib
```

Comment:
```suggestion

```
I don't think anyone cares. Works fine, right?

Reply:
Going to add a comment to use https://github.com/stackbuilders/hspec-golden one day

Reply:
Gold plating? ;)

Reply:
It didn't exist when I wrote this code, but I'd rather have smaller codebase footprint. Very low impact, low priority :)